### ISSUE TITLE
The Gradle build system causes ao to assert

### ISSUE THREAD

**Issue by emanuelez** [boyski/audited-objects#10]
Here's an example:

```
$ ao run ./gradlew assemble
Assertion failed: (path), function pn_new, file ./pn.c, line 94.
Starting a Gradle Daemon (subsequent builds will be faster)
Assertion failed: (path), function pn_new, file ./pn.c, line 94.
Could not write standard input into: Gradle build daemon.
java.io.IOException: Stream closed
        at java.lang.ProcessBuilder$NullOutputStream.write(ProcessBuilder.java:433)
        at java.io.OutputStream.write(OutputStream.java:116)
        at java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)
        at java.io.BufferedOutputStream.flush(BufferedOutputStream.java:140)
        at org.gradle.process.internal.streams.ExecOutputHandleRunner.run(ExecOutputHandleRunner.java:56)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
        at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)

FAILURE: Build failed with an exception.

* What went wrong:
Unable to start the daemon process.
This problem might be caused by incorrect configuration of the daemon.
For example, an unrecognized jvm option is used.
Please refer to the user guide chapter on the daemon at https://docs.gradle.org/3.1/userguide/gradle_daemon.html
Please read the following process output to find out more:
-----------------------


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
> Starting Daemonao: Warning: audit group closed with 2 pending audits
ao: Warning: /private/var/folders/fb/3v_41kkj05lchxcrm1vwvd0c0000gn/T/hsperfdata_emanuelez/4800: No such file or directory
ao: UPLOADING /Users/emanuelez/.gradle/daemon/3.1/registry.bin
ao: UPLOADING /Users/emanuelez/.gradle/daemon/3.1/registry.bin.lock
ao: UPLOADING /Users/emanuelez/.gradle/native/21/osx-amd64/libnative-platform-curses.dylib.lock
ao: UPLOADING /Users/emanuelez/.gradle/native/21/osx-amd64/libnative-platform.dylib.lock
ao: ARCHIVING /Users/emanuelez/.gradle/daemon/3.1/registry.bin.lock
```

**Comment by boyski** [comment]
Some background:
1. I have very little time to give the AO project nowadays. I did the
   original coding during a quiet time in my work life but that time has
   ended. I'd like to give it more time and support but cannot, and it's a
   project of 1 person so no one else will.
2. The server side of AO is deprecated and will not be supported even if I
   find more time. The client side is fairly robust and stable but the Java
   server was never satisfactory, and I came up with a better design which is
   what I was working on when I got too busy. I believe the server situation
   is documented somewhere though I can't remember where.

Bottom line, the client code should work in most cases for what it can do
(general auditing, dependency generation, even makefile regeneration) but
the recycling (aka winkin) feature depends on the server. The old server is
unsupported and the new one doesn't exist yet. I may in some cases be able
to help out with client issues but not with the server at least in the near
term.

David
